libs/common/src/components/screens/welcome/index.tsx

```diff
@@ -101,10 +103,22 @@ export const Welcome = observer<WelcomeProps>(function Welcome({
   const theme = useTheme();
 
   const accountPickerModalProps = useAccountPickerModalProps();
+  const allowedButtons = theme.welcome?.buttons ?? [];
+  const isAllowed = (button: WelcomeButtons) => allowedButtons.includes(button);
```

Comment:
Instead it's probably easier to map over `theme.welcome.buttons`. That way, you wouldn't need all the conditions and it would be possible for themes to change the order of the buttons. See `OsmosisSettings`.